BIDSTools/Experiment.py

```diff
@@ -0,0 +1,37 @@
+import csv as csv
+import json
+
+
+class Experiment:
```

Comment:
please add a description of what is the purpose of this new class (with comments / docstring formatted in a standardized way)